app/domain/authentication/authn_jwt/input_validation/parse_mapped_claims.rb

```diff
@@ -0,0 +1,73 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Parse mapped-claims secret value and return a validated claims list
+      ParseMappedClaims ||= CommandClass.new(
+        dependencies: {
+          validate_claim_name: ValidateClaimName.new(
+            deny_claims_list_value: MANDATORY_CLAIMS_DENY_LIST
+          ),
+          logger: Rails.logger
+        },
+        inputs: %i[mapped_claims_secret_value]
```

Comment:
Same as the other class, i dont need we should call this variable that's way.. mapped_claims should be fine

Reply:
Done